hvplot/plotting/scatter_matrix.py

```diff
@@ -43,7 +46,22 @@ def scatter_matrix(data, c=None, chart='scatter', diagonal='hist',
         Default is `Category10 <https://github.com/d3/d3-3.x-api-reference/blob/master/Ordinal-Scales.md#category10>`.
     diagonal_kwds/hist_kwds/density_kwds: dict, optional
         Keyword options for the diagonal plots
-    kwds: Keyword options for the off-diagonal plots, optional
+    datashade (default=False):
+        Whether to apply rasterization and shading using datashader
+        library returning an RGB object
+    rasterize (default=False):
+        Whether to apply rasterization using the datashader library
+        returning an aggregated Image
+    dynspread (default=False):
+        Allows plots generated with datashade=True or rasterize=True 
+        to increase the point size to make sparse regions more visible.
+        kwds supported include ``max_px``, ``threshold``,  ``shape``, ``how`` and ``mask``.
```

Comment:
```suggestion
        For plots generated with datashade=True or rasterize=True, 
        automatically increase the point size when the data is sparse
        so that individual points become more visible.
        kwds supported include ``max_px``, ``threshold``,  ``shape``, ``how`` and ``mask``.
```